src/pages/api/prisma.tsx

```diff
@@ -0,0 +1,11 @@
+import { PrismaClient } from "@prisma/client";
```

Comment:
change file to .ts as there is no html in this file

Reply:
applied